integrations/pjrt/third_party/pjrt_c_api/CMakeLists.txt

```diff
@@ -26,3 +26,26 @@ iree_cc_library(
     "xla/pjrt/c/pjrt_c_api.h"
   PUBLIC
 )
+
+find_package(Protobuf REQUIRED)
```

Comment:
Not sure if finding a system-wide protobuf is good in IREE. Maybe we should use `FetchContent` to fetch a protobuf and build a `protoc` before generating sources from schema?

Reply:
It's now replaced by `FetchContent` : )